### ISSUE TITLE
For MaskedTextBox with Mask setting, the Name property in Inspect should be empty

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#7938]
### .NET version

8.0.100-alpha.1.22512.32

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Yes, cannot repro in previous .Net versions: 7.0/6.0

### Issue description

For MaskedTextBox with Mask setting, the Name property in Inspect should be empty. This issue is regressed from PR fixing: https://github.com/dotnet/winforms/pull/7741

**Actual result:**
![image](https://user-images.githubusercontent.com/26474449/195556178-52ce34c6-1a6a-44a5-ae23-89d84a2f75dd.png)

**Expected result:**
![image](https://user-images.githubusercontent.com/26474449/195556623-ffa6df7c-0daa-4785-b76a-395ec74a9619.png)

### Steps to reproduce

**Repro steps:**
1.	Create a Winforms .Net 8.0 application
2.	Add several maskedTextBox controls
3.	Set them with different mask: PhoneNumeber, ShortDate
4.	Build and run application, use Inspect to check these controls' Name property

or use this attached sample application:
[WinFormsApp8.zip](https://github.com/dotnet/winforms/files/9774999/WinFormsApp8.zip)

**Comment by merriemcgaw** [comment]
The AccessibleName should be empty to ensure we flag developers to create an appropriate name when they test their accessibility.

**Comment by MelonWang1** [comment]
Verified this on .NET 8.0 latest build: 8.0.100-alpha.1.22552.10, issue was fixed. For MaskedTextBox with Mask setting the Name property in Inspect is empty. 
![7938](https://user-images.githubusercontent.com/94418985/199679801-516fc886-c46b-4491-8fbc-bd30852e8ae4.png)
